Content.Client/Atmos/Visualizers/GasThermoMachineVisualsSystem.cs

```diff
@@ -0,0 +1,47 @@
+using Robust.Client.GameObjects;
+using Content.Shared.Atmos.Piping.Unary.Visuals;
+using Content.Shared.Wires;
+using Content.Client.Wires.Visualizers;
+
+namespace Content.Client.Atmos.Visualizers
+{   /// <summary>
+    /// Controls client-side visuals for gas thermomachines (freezers/heaters).
+    /// </summary>
+    public sealed class GasThermoMachineSystem : VisualizerSystem<GasThermoMachineVisualsComponent>
+    {
+        protected override void OnAppearanceChange(EntityUid uid, GasThermoMachineVisualsComponent visuals, ref AppearanceChangeEvent args)
+        {
+            if (args.Sprite == null)
+                return;
+
+
+            args.AppearanceData.TryGetValue(ThermoMachineVisuals.Running, out var isRunningObject);
```

Comment:
Formatting needs fixing.

Also this should just be using the component rather than an entirely separate visuals component.

Reply:
Yeah, realized that if i scrapped animations the generic visualizer is enough. I'm on it.